core/controllers/profile_test.py

```diff
@@ -224,6 +224,8 @@ class ProfileLinkTests(test_utils.GenericTestBase):
     USERNAME = 'abc123'
     EMAIL = 'abc123@gmail.com'
     PROFILE_PIC_URL = '/preferenceshandler/profile_picture_by_username/'
+    EMAIL_HASH = 'a751156c43e6349d8eae1f392950df34'
+    GRAVATAR_URL = 'http://www.gravatar.com/avatar/'+ EMAIL_HASH+'?d=retro&s=50'
```

Comment:
In a previous comment ("why retro?") I suggested using the identicon instead as a fallback, and storing None if any HTTP (or other) errors occur.